### DISCUSSION TITLE
freq_center - how use? (customing freq into FHSS.cpp)

### DISCUSSION THREAD

**Discussion by Stasmor** [ExpressLRS/ExpressLRS#3001] [Q&A]
Hello all!
Please tell me, about what this attribute/field (freq_center)?
How its must be calculated?
I want modify:
`{"FCC915", FREQ_HZ_TO_REG_VAL(903500000), FREQ_HZ_TO_REG_VAL(926900000), 40, 915000000},` to 
`{"FCC750", FREQ_HZ_TO_REG_VAL(750500000), FREQ_HZ_TO_REG_VAL(770900000), 40, 750000000}` 
will it work, or what i must change for 750 MHz module?
Thank you!

**Comment by JyeSmith** [comment]
ELRS doesnt support any 750M hardware.

**Reply by zaebutcher** [reply]
https://dzen.ru/a/ZmnpPYHU2WoZ0bxV
